accounts/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,33 @@
+<!-- templates/signup.html -->
+{% extends 'fullscreen_header.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'css/login.css' %}">
+{% endblock %}
+
+{% block title %}بازیابی رمز عبور{% endblock %}
+
+{% block inside_content %}
+	<div class="col-lg-6 col-sm-8 mx-auto bg-purple-glass">
+	{% if validlink %}
+    <h2 class="mb-4">یک گذرواژه‌ی جدید انتخاب کنید</h2>
+		{% if messages %}
+			{% for message in messages %}
+				<div class="alert alert-{{ message.tags }}" role="alert">
+					{{ message }}
+				</div>
+			{% endfor %}
+		{% endif %}
+		<form method="post" class="col-9 col-sm-8 mx-auto ltr-form">
+        {% csrf_token %}
+        {{ form | crispy }}
```

Comment:
The form contains a list (of suggestions for how a new password should be) which is half Persian half English, and Crispy turns it into a color that is barely readable. @kiarashgl can you take care of these?

Reply:
Okay, for sure :grinning: